.github/ci-cd-scripts/playwright-browser-chrome-linux.sh

```diff
@@ -0,0 +1,42 @@
+if [[ ! -f "test-results/.last-run.json" ]]; then
```

Comment:
Every bash script should start with the following. It's like the Bash equivalent of JS's `'use strict'`.

```bash
set -euo pipefail
```